pkgs/development/python-modules/onnxconverter-common/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, protobuf, numpy, requests, onnx }:
+
+buildPythonPackage rec {
+  pname = "onnxconverter_common";
+  version = "1.7.0";
+
+  propagatedBuildInputs = [ protobuf numpy requests onnx ];
+  format = "wheel";
+  src = fetchPypi {
+    inherit pname version;
+    format = "wheel";
+    sha256 = "1mgfxvcii4h6d7z2zbr5979aajhdblayfpaakm0vv9ka313i0dvd";
+  };
```

most common ordering of attrs
```suggestion
  format = "wheel";
  
  src = fetchPypi {
    inherit pname version;
    format = "wheel";
    sha256 = "1mgfxvcii4h6d7z2zbr5979aajhdblayfpaakm0vv9ka313i0dvd";
  };
    
  propagatedBuildInputs = [ protobuf numpy requests onnx ];
```